src/tests.c

```diff
@@ -3101,14 +3101,17 @@ void test_ge(void) {
 
     /* Test batch gej -> ge conversion with many infinities. */
     for (i = 0; i < 4 * runs + 1; i++) {
+        int odd;
         random_group_element_test(&ge[i]);
+        odd = secp256k1_fe_is_odd(&ge[i].x);
+        CHECK(odd == 0 || odd == 1);
         /* randomly set half the points to infinity */
-        if(secp256k1_fe_is_odd(&ge[i].x)) {
+        if (odd == i % 2) {
```

Comment:
Nit: `% 2` seems superfluous here.

Reply:
Ah I guess it depends on the thing you want to test. 

Before the change, the input consisted of some infinity points and some affine points *with even x coordinate*.
After the change, the input consists of some infinity points and some (arbitrary) affine points.

Reply:
Well the line before is `CHECK(odd == 0 || odd == 1);`. The `% 2` is guaranteed to not have any effect.

Reply:
Nevermind, I was just misreading this line as `if (odd % 2) {`.